pypeit/reduce.py

```diff
@@ -211,7 +213,7 @@ def initialise_slits(self, initial=False):
 
         # Slitmask
         self.slitmask = self.slits.slit_img(initial=initial, flexure=self.spat_flexure_shift,
-                                            exclude_flag=self.slits.bitmask.exclude_for_reducing)
+                                            exclude_flag=self.slits.bitmask.exclude_for_reducing+['BOXSLIT'])
```

Comment:
I'm confused by the syntax here. What is + ['BOXSLIT'] doing?

Reply:
Okay I think I understand the syntax now. Sorry was confused but now realize that you are just adding two lists.